Archit/Source/Films/FilmCell.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  FilmCell.swift
+//  Archit
+//
+//  Created by Diego Manuel Molina Canedo on 11/12/17.
+//  Copyright © 2017 Intelygenz. All rights reserved.
+//
+
+import UIKit
+
+class FilmCell: UITableViewCell {
+
+    @IBOutlet weak private var posterImageView: UIImageView!
+    @IBOutlet weak private var titleLabel: UILabel!
+    @IBOutlet weak private var yearLabel: UILabel!
+    
+    override func awakeFromNib() {
```

Comment:
If you don't use it, you can delete this method.

Reply:
Removed